index.js

```diff
@@ -224,9 +224,12 @@ module.exports = {
 
     self.mergeProfile = function(profile, user) {
       _.each(profile, function(val, key) {
+        // Check if a property should not be updated from an existing value.
+        var excludeKey = _.includes(options.avoidOverride, key) && user[key] !== undefined;
+
         // Do not clobber email entered in Apostrophe just because
         // none was defined in the profile
-        if (val !== undefined) {
+        if (val !== undefined && !excludeKey) {
```

Comment:
This all looks good to me, but I'd like to see more use of `()` here. I am shaky on who goes first, `!==` or `&&` so I doubt a less experienced reader would be.